go/kbfs/libkbfs/kbfs_ops_test.go

```diff
@@ -1373,81 +1470,88 @@ func makeDirTree(id tlf.ID, uid keybase1.UserOrTeamID, components ...string) (
 
 	bid := makeBlockID()
 	bi := makeBIFromID(bid, uid)
-	rootEntry := DirEntry{
+	rootEntry := data.DirEntry{
 		BlockInfo: bi,
-		EntryInfo: EntryInfo{
-			Type: Dir,
+		EntryInfo: data.EntryInfo{
+			Type: data.Dir,
 		},
 	}
-	nodes := []pathNode{{bi.BlockPointer, "{root}"}}
-	rootBlock := NewDirBlock().(*DirBlock)
+	nodes := []data.PathNode{{BlockPointer: bi.BlockPointer, Name: "{root}"}}
+	rootBlock := data.NewDirBlock().(*data.DirBlock)
 	rootBlock.SetEncodedSize(bi.EncodedSize)
-	blocks := []*DirBlock{rootBlock}
+	blocks := []*data.DirBlock{rootBlock}
 
 	// Handle the rest.
 
 	parentDirBlock := rootBlock
 	for _, component := range components {
 		bid := makeBlockID()
 		bi := makeBIFromID(bid, uid)
-		parentDirBlock.Children[component] = DirEntry{
+		parentDirBlock.Children[component] = data.DirEntry{
 			BlockInfo: bi,
-			EntryInfo: EntryInfo{
-				Type: Dir,
+			EntryInfo: data.EntryInfo{
+				Type: data.Dir,
 			},
 		}
-		nodes = append(nodes, pathNode{bi.BlockPointer, component})
-		dirBlock := NewDirBlock().(*DirBlock)
+		nodes = append(nodes, data.PathNode{
+			BlockPointer: bi.BlockPointer,
+			Name:         component,
+		})
+		dirBlock := data.NewDirBlock().(*data.DirBlock)
 		dirBlock.SetEncodedSize(bi.EncodedSize)
 		blocks = append(blocks, dirBlock)
 
 		parentDirBlock = dirBlock
 	}
 
-	return rootEntry, path{FolderBranch{Tlf: id}, nodes}, blocks
+	return rootEntry, data.Path{
+			FolderBranch: data.FolderBranch{Tlf: id},
+			Path:         nodes},
```

Comment:
I'd prefer this closing brace on a new line. In fact, each element in the return can be on its own line.